docs/Makefile

```diff
@@ -25,7 +25,8 @@ setupenv:
 setup:
 	$(POETRY) install
 	$(POETRY) update
-	cp -TLr source $(SOURCEDIR)
+	@if [ ! -d "$(SOURCEDIR)" ]; then mkdir -p "$(SOURCEDIR)"; fi
```

Comment:
`@` prevents `make` from printing it to stdout when it is executed, right? Why did you decide to silence it? Wouldn't it be better for the sake of transparency to print this line?

Also, isn't the `if` redundant here? AFAIK `mkdir -p` does nothing if the directory exists.

Reply:
@dgarcia360

Reply:
Addressed in https://github.com/scylladb/scylla-rust-driver/pull/927/commits/6f11d339e2b1c0e653524cc0153423dfddd162e9